sale_partner_insurance/models/res_partner.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Akretion France (http://www.akretion.com/)
+# @author: Mathieu Delva <mathieu.delva@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    sale_insurance = fields.Float()
```

Comment:
coefficient_sale_insurance will be a more explicit name, also a "help" will be nice. Same on the other field in Company / settings.